includes/Modules/Idea_Hub.php

```diff
@@ -101,6 +100,21 @@ function( $post_states, $post ) {
 				10,
 				2
 			);
+
+			/**
+			 * Allows us to trash / modify empty idea posts.
+			 */
+			add_filter(
+				'wp_insert_post_empty_content',
+				function( $maybe_empty, $postarr ) {
```

Comment:
Is `$postarr` WordPress convention? If so, I'm fine with keeping it (respecting convention is good!). But otherwise it's a bit of an ambiguous variable name to me. I'm guessing it means "post array" eg "post, accessible as an array", in which case `$post_as_array` isn't much more to write but is a lot easier to instantly parse 😄 

Equally, _what_ is `$maybe_empty` here? Is it a post? The contents of the post?

Reply:
Update: I can see in the docs these are the variables this filter uses: https://developer.wordpress.org/reference/hooks/wp_insert_post_empty_content/

Not great names in my mind, but sticking with convention is good! 😅